src/matlab2tikz.m

```diff
@@ -2724,7 +2728,8 @@ function assertRegularAxes(handle)
         cl = 'specgraph.errorbarseries';
 
     % quiver plots
-        % TODO: maybe check for existence of `udata` and absence of `ldata`
+    elseif hasProperties(h, {'udata','vdata'}, {'ldata'})
```

Comment:
Are you sure? Do note that this function is Octave-only! Normally, LData is used to store the lower errorbar, which we don't have on quivers (UData is either used for the upper errorbar or the x-direction of the arrow).

In Octave, it seems that each of the HG objects only have the relevant properties present, unlike in HG1 in MATLAB where all properties are present (HG2 is a bit more strict, but I don't know whether quivers actually do or do not have the LData in HG2).